salt/_modules/caasp_orch.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+
+def __virtual__():
+    return "caasp_orch"
+
+
+def sync_all():
+    '''
+    Syncronize everything before starting a new orchestration
+    '''
+    __utils__['caasp_log.debug']('orch: synchronizing the mine')
+    __salt__['saltutil.runner']('mine.update', tgt='*', clear=True)
+
+    # TODO: maybe sync modules and so...
```

Comment:
i was thinking, how much is the effort to sync all modules? this could spare some error msg we have in minions, we could maybe do it in the same pr if it is 3 lines